app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::CommentsController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    new_id = params[:post_id]
+    @comments = Comment.where({ post_id: new_id }).order(:created_at)
+    render json: { data: { comments: @comments } }, status: :ok
+  end
```

Comment:
- [ ] I can't access comments of my post. 

![Screenshot 2021-12-16 at 21-06-26 Action Controller Exception caught](https://user-images.githubusercontent.com/1642014/146426275-791fa3a0-281b-4dbc-9bf2-9dee238287e0.png)